devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy

```diff
@@ -44,8 +44,8 @@ metadata {
 		command "updateWeather"
 
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "MaestroStat", deviceJoinName: "Stelpro Maestro Thermostat"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "SORB", deviceJoinName: "Stelpro Maestro Thermostat"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "SonomaStyle", deviceJoinName: "Stelpro Maestro Thermostat"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "SORB", deviceJoinName: "Stelpro Orleans Fan-Heater"
```

Comment:
Is there supposed to be a dash between Fan and Heater? The website doesn't have one: https://www.stelpro.com/en-US/sonoma-style-wall-fan-heater

Reply:
Those are the values that I received from @stelpro . Also, realized that I had left out "Style" from the Sonoma string, so I just updated that.